saleor/registration/views.py

```diff
@@ -68,3 +83,44 @@ def password_reset_confirm(request, uidb64=None, token=None):
         'uidb64': uidb64}
     return PasswordResetConfirm.as_view(**kwargs)(
         request, **kwargs)
+
+
+class EmailVerificationView(View):
```

Comment:
We don't use class-based views, could you please change this view to function? :) Also, main functionality could be enclosed in form, e.g. ```EmailVerificationForm```.